src/js/components/block.js

```diff
@@ -77,24 +138,45 @@ export class FormBlock extends Component {
       props.footer = footer;
     }
 
+    const {zDepth, className, thinActionButton} = this.props;
+    if ( !isUndefined( zDepth ) ) {
+      props.zDepth = zDepth;
+    }
+    if ( isString( className ) ) {
+      props.className = className;
+    }
+
+    /*eslint-disable no-unused-vars*/
```

Comment:
Eslint does not yet support a way to detect that <ButtonClass> is a usage of const ButtonClass, triggering the no-unused-vars rule.

The no-unused-vars rule is *extremely* helpful elsewhere, to detect unused imports and props. It makes more sense to use the /*eslint-disable*/ tags in a few places to keep the rule activated everywhere else.